reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/utils/ApplicationConfiguration.java

```diff
@@ -71,4 +80,70 @@ public class ApplicationConfiguration {
     @Value("${adapter.master.project.search.path}")
     private String adapterMasterProjectSearchPath;
 
+    @Value("${mgramseva.host}")
+    private String mgramsevaHost;
+
+    @Value("${mgramseva.oauth.access.token.url}")
```

Comment:
Are these endpoints or complete urls(consisting hostname)? 
If they are endpoints, replace url suffix with endpoint.

Reply:
These are endpoints. But we have more attribute, which are being followed same.
for endpoint suffix is URL, for host suffix is host. i.e. ifix.host

Should I change all or just new implementations.

Reply:
Change all.

Reply:
sure

Reply:
done